### PR TITLE
Fix roles language in docs

### PR DESCRIPTION
**Pull request by samdoran**
##### SUMMARY

Correct docs on how tags work on roles.

##### ISSUE TYPE

 - Docs Pull Request

##### COMPONENT NAME

Docs

 ##### ANSIBLE VERSION

```
2.3.1.0
```

##### ADDITIONAL INFORMATION
Tags added at the role level are appended to task in a role and do not replace them.

### CODE REVIEW

File: docs/docsite/rst/playbooks_reuse_roles.rst [hunk 1]
```diff
@@ -170,14 +170,14 @@ Or, again, using the newer syntax::
...
-      - import_role: 
+      - import_role:
           name: foo
         tags:
         - bar
         - baz
 
 .. note::
-    This *tags all of the tasks in that role with the tags specified*, overriding any tags that are specified inside the role. If you find yourself building a role with lots of tags and you want to call subsets of the role at different times, you should consider just splitting that role into multiple roles.
+    This *tags all of the tasks in that role with the tags specified*, appending to tags that are specified inside the role. If you find yourself building a role with lots of tags and you want to call subsets of the role at different times, you should consider just splitting that role into multiple roles.
```
Comment (bcoca) [inline docs/docsite/rst/playbooks_reuse_roles.rst]: this should be true for import, but include shoudl have tags applied to IT

Comment (samdoran) [reply docs/docsite/rst/playbooks_reuse_roles.rst]: Not sure exactly what you mean by import vs include. So it's true for `roles` but not `include_role`?

Comment (bcoca) [reply docs/docsite/rst/playbooks_reuse_roles.rst]: `roles` and `import_role` but not `include_role`

Comment (dharmabumstead) [reply docs/docsite/rst/playbooks_reuse_roles.rst]: Please amend to "any tags that are already specified inside the role". Thanks!

Comment (samdoran) [reply docs/docsite/rst/playbooks_reuse_roles.rst]: I see: `import_role` is new. Much more explicit dynamic/static includes. Nice! Let me try to add that to the note without overcomplicating it.

File: docs/docsite/rst/playbooks_reuse_roles.rst [hunk 2]
```diff
@@ -170,14 +170,14 @@ Or, again, using the newer syntax::
...
-      - import_role: 
+      - import_role:
           name: foo
         tags:
         - bar
         - baz
 
 .. note::
-    This *tags all of the tasks in that role with the tags specified*, overriding any tags that are specified inside the role. If you find yourself building a role with lots of tags and you want to call subsets of the role at different times, you should consider just splitting that role into multiple roles.
+    This *tags all of the tasks in that role with the tags specified*, appending to any tags that are specified inside the role. The tags in this example will *not* be added to tasks inside an ``include_role``. Tag the ``include_role`` task directly in order to apply tags to tasks in included roles. If you find yourself building a role with lots of tags and you want to call subsets of the role at different times, you should consider just splitting that role into multiple roles.
```
Comment (samdoran) [inline docs/docsite/rst/playbooks_reuse_roles.rst:180]: This is rather tough to explain succinctly. Let me know if this is way too opaque.

**Comment by samdoran** [comment]
Any objections before merging?

**Review by dharmabumstead — APPROVED** [review]

**Comment by dharmabumstead** [comment]
Thanks @samdoran!

**Pull request merged**